fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+      The response will contain the partially calculated draft order and any calculated attributes dependant on stepwise operations will be null/empty until available.
+      
+      For long running operations, clients will be responsible for polling the endpoint specified in the Location header until a 200 OK response is returned.
+      
+    x-traitTag: true  
+    
+    
+
+paths:
+  /admin/draft_orders.json:
+    get:
+      summary: Retrieve a list of draft orders
+      description: Return a list of draft order objects.
+      parameters:
+      - $ref: '#/parameters/fields'
+      - $ref: '#/parameters/limit'
+      - $ref: '#/parameters/page'
+      - $ref: '#/parameters/since_id'
+      - $ref: '#/parameters/status'
+      - $ref: '#/parameters/ids'
+    
+      tags:
+        - Get
+
+      responses:
+        '200':
+          description: List of draft orders
+          schema:
+            $ref: '#/definitions/DraftOrder'
+        
+        '400':
+          description: Bad request
+          schema:
+            type: object
+            properties:
+              errors:
+                  $ref: '#/definitions/Error'
+                 
+       
+          
+            
+      x-code-samples:
+        - lang: curl
+          source:  |
+            curl -X GET -H "Content-Type: application/json" https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin/draft_orders.json
+            
+        
+            
+      
+        - lang: Ruby
+          source: |
+              require 'shopify_api'
+
+              shop_url = "https://#{api_key}:#{api_password}@your_store_domain.myshopify.com/admin"
+              ShopifyAPI::Base.site = shop_url
+              draft_order = ShopifyAPI::DraftOrder.all
+            
+            
+  
+    post:
+      summary: Create a draft order
+      description: |
+          
+          Using the Draft Orders API you can create orders in draft state using product variant line items, or your own custom line items.  This enables you to create one-off products that exist outside of your regular inventory management system.  To create a product variant draft order, simply specify a `variant_id` `quantity` and `discount` (if applicable).  Create a custom line item by supplying `title`, `price`, `taxable` and `quantity`. 
+          
+        
+          | `Note`| The array of line items on a draft order must not contain a `variant_id` more than once. The data copied over from a variant upon creation of a line item is never subsequently updated.  |
+          |---|-------------|
+          
+          #### About Custom Shipping Lines
+          
+            The Draft Orders API provides you with the flexibility of sending orders to your customers using custom shipping lines.  A custom shipping line includes a `title` and `price` with `handle` set to Nil.  A shipping line with a carrier provided shipping rate (currently only set via the admin) includes the shipping rate `handle`.
```

Comment:
* re: "provides the flexibility of sending orders.. etc." -- we tend to avoid this more promotional tone -- consider some more modest phrasing such as "You can use the Draft Orders API to send orders to customers using custom shipping lines." (if this is a major selling feature, then we should be promoting it in the paragraph(s) at the very top of the page) 
* I'm noticing `Nil` here (and at line 979), but "null" elsewhere -- we should standardize those if they're the same thing.
* "carrier provided shipping rate" -> "carrier-provided shipping rate"
* "via the admin" -> "via the Shopify admin"